uaclient/messages.py

```diff
@@ -731,6 +735,14 @@ class TxtColor:
     "Cannot install Real-Time Kernel on a container.",
 )
 
+GCP_SERVICE_ACCT_NOT_ENABLED_ERROR = NamedMessage(
+    "gcp-pro-service-account-not-enabled",
+    "Failed to attach machine\n" + "{error_msg}.",
```

Comment:
Not sure if we need the `.` and no need for concatenating things
```suggestion
    "Failed to attach machine\n{error_msg}",
```